code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -202,7 +202,7 @@
 		observer.forceMove(O.loc)
 	else
 		to_chat(src, "<span class='notice'>Teleportacja nieudana. Poinformuj admina używając Ahelp</span>")
-		stack_trace("Na mapie brakuje observer mark lub próbujesz obserwować zanim gra się zaczeła") //co?

```

Comment:
stack trace, ASSERT i CRASH to crashe w kodzie, tych wiadomości nie tłumacz, info dla adminów to jest

Reply:
jak tak chcesz, admini też mogą mieć po polsku